### PR TITLE
updates gift empire to v0.6

### PR DESCRIPTION
**Pull request by gabeio**

### CODE REVIEW

**Comment by gabeio** [comment]
Screen Shot 2021-06-02 at 01 22 44


and I actually had the chance to test this, it does work.

**Comment by jacobrosenthal** [comment]
Thanks! Can we get the v06 label and a date bump?

File: content/diplomacy/gift-empire/plugin.js [hunk 1]
```diff
@@ -144,4 +144,4 @@ class Plugin {
   }
 }
 
-plugin.register(new Plugin());
+export default plugin;
```
Comment (jacobrosenthal) [inline content/diplomacy/gift-empire/plugin.js:147]:
```suggestion
export default Plugin;
```

Comment (gabeio) [reply content/diplomacy/gift-empire/plugin.js:147]: 🤦 I was doing this late at night, thanks for the catch.

**Comment by gabeio** [comment]
@jacobrosenthal all suggested changes have been made.

**Pull request merged**